live/src/main/java/org/dbpedia/extraction/live/queue/LiveQueue.java

```diff
@@ -23,7 +23,7 @@
 
     private static final PriorityBlockingQueue<LiveQueueItem> queue = new PriorityBlockingQueue<LiveQueueItem>(1000);
     // this is a Map<long,int>, long is the pageID and int the number of same items in the queue (see add())
-    private static final Set<Long> uniqueSet = new HashSet(1000);
+    private static final Set<String> uniqueSet = new HashSet(1000);
```

Comment:
IIRC, Before RCStream not all items had a title, now not all items have a pageID so I am not sure if this change will work